cmd/buildah/commit.go

```diff
@@ -40,6 +40,10 @@ var (
 			Usage: "`format` of the image manifest and metadata",
 			Value: "oci",
 		},
+		cli.StringFlag{
```

Comment:
buildah commit and docker commit are somewhat different but I think we should add this to podman commit, since if it has value for build it probably has value to commit.